### PR TITLE
CMRARC-635 Add GHRC_DAAC to list of GHRC provider

### PR DESCRIPTION
**Pull request by htranho**

### CODE REVIEW

**Review by william-valencia — APPROVED** [review]

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -126,21 +126,21 @@ def self.truncate_string(string, max)
...
-    providers = %w[GHRC_CLOUD LANCEAMSR2] if virtual_daac == 'GHRC'
+    providers = %w[ASDC LARC_ASDC LARC] if virtual_daac == 'ASDC'
+    providers = %w[NSIDC NSIDCV0 NSIDC_ECS] if virtual_daac == 'NSIDC'
+    providers = %w[GHRC GHRC_CLOUD GHRC_DAAC LANCEAMSR2] if virtual_daac == 'GHRC'
     providers
   end
 
   def self.all_arc_providers
-    return ARC_PROVIDERS + %w(LARC_ASDC LARC NSIDCV0 NSIDC_ECS GHRC GHRC_CLOUD LANCEAMSR2)
+    return ARC_PROVIDERS + %w(LARC_ASDC LARC NSIDCV0 NSIDC_ECS GHRC GHRC_CLOUD GHRC_DAAC LANCEAMSR2)
```
Comment (cgokey) [inline app/helpers/application_helper.rb:143]: Shouldn't this include ASDC NSIDC and GHRC as well?

File: app/models/cmr.rb [hunk 2]
```diff
@@ -506,12 +506,9 @@ def self.collection_search(free_text, provider, provider_list, curr_page="1", pa
       #setting the provider params
       if provider == ANY_DAAC_KEYWORD
         base_options['provider'] = provider_list + ['LARC_ASDC', 'LARC'] +
-          ['NSIDCV0', 'NSIDC_ECS'] + ['GHRC', 'GHRC_CLOUD', 'LANCEAMSR2']
+          ['NSIDCV0', 'NSIDC_ECS'] + ['GHRC', 'GHRC_CLOUD', 'GHRC_DAAC', 'LANCEAMSR2']
```
Comment (cgokey) [inline app/models/cmr.rb:509]: Shouldn't this include ASDC NSIDC and GHRC as well?

File: test/features/can_search_collection_test.rb [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+      with(
+        headers: {
+          'Accept'=>'*/*',
+          'Accept-Encoding'=>'gzip;q=1.0,deflate;q=0.6,identity;q=0.3',
+          'User-Agent'=>'Ruby'
+        }).
+      to_return(status: 200, body: get_stub("search_collection_C1996546695-GHRC_DAAC.atom"), headers: {})
+  end
+  describe 'search cmr collection' do
+    it 'search collection by provider' do
```
Comment (cgokey) [inline test/features/can_search_collection_test.rb:47]: Maybe name it, "search collection by GHRC provider"

File: app/helpers/application_helper.rb (hunk 1 above)
Comment (htranho) [reply app/helpers/application_helper.rb]: now included

File: app/models/cmr.rb (hunk 2 above)
Comment (htranho) [reply app/models/cmr.rb]: Now included

File: test/features/can_search_collection_test.rb (hunk 3 above)
Comment (htranho) [reply test/features/can_search_collection_test.rb]: Changed

**Review by cgokey — APPROVED** [review]

**Pull request merged**